### PR TITLE
Add privacy notice

### PR DESCRIPTION
**Pull request by lesleyjanenorton**

### CODE REVIEW

File: extension/js/background.js [hunk 1]
```diff
@@ -41,9 +41,17 @@ async function getOrMakeGAUUID() {
 
 
 async function sendMetricsEvent(eventData) {
-  if (navigator.doNotTrack === "1") {
+  const doNotTrackIsEnabled = (navigator.doNotTrack === "1");
+  const { dataCollection } = await browser.storage.local.get("dataCollection");
+
+  if (!dataCollection) {
+    browser.storage.local.set({ "dataCollection": "data-enabled" });
+  }
```
Comment (groovecoder) [inline extension/js/background.js:49]: This means opt-out, so we'll have to make sure the approved content also indicates opt-out.

File: extension/js/first-run.js [hunk 2]
```diff
@@ -1,6 +1,69 @@
...
+  if (!dataCollection) {
+    setTimeout(() => {
+      openModal();
+    }, 500);
+  }
+
+
+
+
+
```
Comment (groovecoder) [inline extension/js/first-run.js:66]:
```suggestion
```

File: extension/first-run.html [hunk 3]
```diff
@@ -9,6 +9,27 @@
...
+    <div class="privacy-modal modal hide-modal">
+      <div class="modal-content">
+        <h1 class="modal-h1">Privacy</h1>
+        <div class="modal-copy">
+          <p>May we collect the following anonymous data about your use of Firefox Relay?</p>
+          <ul class="collected-data-list">
+            <li class="data-list-item">When the add-on pop-up is viewed</li>
+            <li class="data-list-item">When a button or link in the add-on is clicked</li>
+            <li class="data-list-item">When the Relay icon is injected into website forms</li>
+            <li class="data-list-item">When a new alias is created</li>
```
Comment (pdehaan) [inline extension/first-run.html:21]: Apologies, I haven't been following all the content changes in this repo, but is this "alias" or "address" these days?

Comment (groovecoder) [reply extension/first-run.html]: "alias" is the latest hotness.

**Pull request merged**